shell/golangci-lint.sh

```diff
@@ -53,7 +53,9 @@ if [[ -z $GOGC ]] && [[ -z $GOMEMLIMIT ]]; then
   # This is mostly important for CI systems or other container
   # environments.
   if (command -v free || command -v sysctl) &>/dev/null; then
-    if command -v free &>/dev/null; then
+    if [ -f /sys/fs/cgroup/memory/memory.limit_in_bytes ]; then
+      mem=$(($(cat /sys/fs/cgroup/memory/memory.limit_in_bytes) / 1024 / 1024))
```

Comment:
suggestion: it occurred to me that this doesn't seem to be related to `free` or `sysctl` being available, so this `if` statement could be on the same level as line 55, instead of being nested inside of line 55.